frontend/hub/collections/CollectionPage/documentationComponents/CollectionDocumentationTabContent.tsx

```diff
@@ -0,0 +1,578 @@
+  const queryString = queryStringFromObject(searchParams);
+  const getPageUrl = useGetPageUrl();
+  const settings = useSettings();
+  const [paramsFilter, setParamsFilter] = useState('');
+  const [error, setError] = useState('');
+  const [json, setJson] = useState('');
+
+  const contentKey = content?.content_type + ' > ' + content?.content_name;
+
+  useEffect(() => {
+    setJson('');
+  }, [content?.content_name, content?.content_type]);
+
+  useEffect(() => {
+    const options: OptionRecord[] = [];
+
+    /*
+      Build parameters options lists. Transform tree (options and suboptions) into one list
+      that can be rendered into table.
+
+      Every item will have info about its full path from first parent to itself.
+    */
+    function fillOptions(
+      local_options: IContentsOption[],
+      level: number,
+      global_options: OptionRecord[],
+      path_name: string
+    ) {
+      if (level === undefined) {
+        return;
+      }
+
+      if (!local_options) {
+        return;
+      }
+
+      local_options.forEach((option) => {
+        let new_path = path_name;
+        if (new_path.length > 0) {
+          new_path += '/';
+        }
+        new_path += option.name;
+
+        // fill boolean choices if their are missing
+        if (option.type === 'bool' && !(option.choices && option.choices.length > 0)) {
+          option.choices = [];
+          option.choices.push('true');
+          option.choices.push('false');
+        }
+        global_options?.push({
+          option,
+          level,
+          path_name: new_path,
+          parent_path: path_name,
+          visible: true,
+          checked: true,
+          children: option.suboptions ? true : false,
+        });
+        if (option.suboptions) {
+          fillOptions(option.suboptions, level + 1, global_options, new_path);
+        }
+      });
+    }
+
+    try {
+      fillOptions(content?.doc_strings?.doc?.options || [], 0, options, '');
+      setOptionsState(options);
+    } catch (ex) {
+      setError('error');
+    }
+  }, [contentKey, content?.doc_strings?.doc?.options]);
+
+  /* Shows/Hides all children of the selected option  */
+  function setVisibility(path: string, value: boolean) {
+    const newState = JSON.parse(JSON.stringify(optionsState)) as OptionRecord[];
+
+    const currentOption = newState.find((item) => item.path_name === path);
+    if (currentOption) {
+      currentOption.checked = value;
+    }
+
+    newState.forEach((optionRecord) => {
+      // search for all record starting with this path
+      if (optionRecord.path_name.startsWith(path)) {
+        // except the exact path, this we dont want to hide
+        if (optionRecord.path_name !== path) {
+          optionRecord.visible = value;
+          optionRecord.checked = value;
+        }
+      }
+    });
+
+    setOptionsState(newState);
+  }
+
+  const backtoMenuLink = <a href={`?${queryString}#Menu_part`}>{t('Back to overview')}</a>;
```

Comment:
```suggestion
  const backtoMenuLink = <a href="#Menu_part">{t('Back to overview')}</a>;
```

(and no need for queryStringFromObject, right?)